jane/doc/extensions/_04-modes/intro.md

```diff
@@ -183,5 +183,64 @@ they are once.
 More documentation on uniqueness and linearity is available
 [here](../../uniqueness/intro/).
 
-<!-- CR ccasinghino: Sections needed for statefulness, visibility, and yielding -->
+## Future modes: Yielding
```

Comment:
This appears under the heading about aliasing; I think that's probably not what you want.

Reply:
Fixed.